docs/API.md

```diff
@@ -570,6 +570,7 @@ By default, with [balenaOS 2.82.6](https://github.com/balena-os/meta-balena/blob
 			"password": "password",
 			"noProxy": [ "152.10.30.4", "253.1.1.0/16" ]
 		},
+		"dns": "1.1.1.1:53",
```

Comment:
This should be part of the proxy block no? It is the dns configuration for the proxy after all, not for the device network